resolver/resolver.cc

```diff
@@ -1653,11 +1665,6 @@ class ResolveTypeMembersAndFieldsWalk {
             return false;
         }
         if (job.lhs.isTypeMember()) {
-            auto superclass = job.lhs.data(ctx)->owner.data(ctx)->superClass();
-            if (!isGenericResolved(ctx, superclass)) {
-                return false;
-            }
-
```

Comment:
Why are we OK to delete this?  Is the generic resolution handled by declaring the superclass as a dependency, below, and thus this check is redundant?  Relatedly, I think we can run parts of resolver multiple times, so is it possible that we can come through here without the generic bits being resolved, despite declaring the superclass as a dependency?

And if the answer to the latter question is "yes", wouldn't we have to do the same for mixins now?

Reply:
It's ok to delete this because the check was moved into the dependencies vector instead. Checking the superclass was sort of redundant work, given that putting it in the dependencies vector would allow us to potentially check it fewer times.